tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -116,15 +78,9 @@ Add meta guidance to subject
     ...  ${SUBJECT_NAME} meta guidance content
     user clicks button  Save guidance
 
-Navigate to 'Create data block' page
+Navigate to Data blocks page
     [Tags]  HappyPath
-    user clicks link    Data blocks
-
-    user waits until h2 is visible  Data blocks
-    user clicks link  Create data block
-
-    user waits until h2 is visible  Create data block
-    user waits until table tool wizard step is available    Choose a subject
+    user navigates to Data blocks section 
```

Comment:
`user navigates to Data blocks section` doesn't click the `Create data block` link, so I believe this will currently fail when waiting for the subject name to appear in the next test case